tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -913,3 +913,169 @@ Feature: List a sharing permissions
       | space        | Personal |
       | sharee       | Brian    |
       | shareType    | user     |
+
+  @issues-8351
+  Scenario: user lists permissions of a project space using root endpoint
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    When user "Alice" lists the permissions of drive "new-space" using root endpoint of the Graph API
```

Comment:
```suggestion
    When user "Alice" lists the permissions of space "new-space" using root endpoint of the Graph API
```

Reply:
but our root endpoint API is using drive word as references in the document. I don't want to create confusion for future maintainer